docs/cloud/moving-into.md

```diff
@@ -0,0 +1,55 @@
+## When to migrate to K8s
+
+When considering migrating your services from Pouta to k8s, it’s important to consider the relative
+strengths of each platform. Both Pouta and Kubernetes-based platforms, such as Rahti, are Cloud
+Computing platforms, but there are some important differences. Virtual machine based services, like
+cPouta, are classified as Infrastructure-as-a-Service (IAAS) cloud services. Kubernetes on the other
+hand provides a container cloud orchestration service. You can deploy services just by using
+Kubernetes, but typically it provides building blocks for developer platforms such as Rahti or
+OpenShift OKD, which are a Platform-as-a-Service (PAAS) systems.
+
+The primary difference between IAAS and PAAS services is that with IAAS the user is responsible
+for managing the operating system and potentially multiple applications whereas with PAAS, the
+user is only responsible for managing the application and the data. You can read more about cloud
+computing concepts [here](https://docs.csc.fi/cloud/concepts/).
```

Comment:
```suggestion
[computing concepts](../concepts/).
```
Minor thing, internal links to docs.csc.fi should be relative links and cannot contain `https://docs.csc.fi/`. Also links should avoid general words like `here`. (I guess I have done this before as well, but little by little we can improve).